packages/scene-graph-mediator/runtime/test/importer/Pixi.js

```diff
@@ -452,5 +461,356 @@ describe('Pixi', () => {
+        }
+      };
+      
+      const templateLayoutParentNode = {
+        constructorName: 'Container',
+        id:   'templateLayoutTestName',
+        name: 'templateLayoutTestName',
+        transform: {
+          x: 0,
+          y: 0,
+          width: 100,
+          height: 100,
+          scale: {
+            x:1.0,
+            y:1.0,
+          },
+          anchor: {
+            x: 0.5,
+            y: 0.5
+          },
+          children: []
+        },
+        layout: {
+          layoutSize: {
+            width: 100,
+            height: 100
+          },
+          resize: 0,
+          layoutType: 1,
+          cellSize: {
+            width: 40,
+            height: 40
+          },
+          startAxis: 0,
+          paddingLeft: 0,
+          paddingRight: 0,
+          paddingTop: 0,
+          paddingBottom: 0,
+          spacingX: 0,
+          spacingY: 0,
+          verticalDirection: 1,
+          horizontalDirection: 0
+        }
+      };
+
+      let layoutChildNode00 = JSON.parse(JSON.stringify(templateLayoutChildNode));
+      layoutChildNode00.id = layoutChildNode00.name = "layoutChild00";
+      layoutChildNode00.transform.parent = 'layoutParentName';
+      let layoutChildNode01 = JSON.parse(JSON.stringify(templateLayoutChildNode));
+      layoutChildNode01.id = layoutChildNode01.name = "layoutChild01";
+      layoutChildNode01.transform.parent = 'layoutParentName';
+      let layoutChildNode02 = JSON.parse(JSON.stringify(templateLayoutChildNode));
+      layoutChildNode02.id = layoutChildNode02.name = "layoutChild02";
+      layoutChildNode02.transform.parent = 'layoutParentName';
+      let layoutChildNode03 = JSON.parse(JSON.stringify(templateLayoutChildNode));
+      layoutChildNode03.id = layoutChildNode03.name = "layoutChild03";
+      layoutChildNode03.transform.parent = 'layoutParentName';
+
+      let layoutParentNode = JSON.parse(JSON.stringify(templateLayoutParentNode));
+      layoutParentNode.id = layoutParentNode.name = 'layoutParentName';
+      layoutParentNode.transform.children = [ layoutChildNode00.name, layoutChildNode01.name, layoutChildNode02.name, layoutChildNode03.name ];
+
+      let schema = {
+        scene: [ layoutParentNode, layoutChildNode00, layoutChildNode01, layoutChildNode02, layoutChildNode03 ],
+        metadata: metadata
+      };
+
+      describe('when layout component is given', () => {
+        
+        before(async()=>{
+          //  FIXME:
+          //  Measures to avoid the problem that sprite size becomes default size (1, 1).
+          //  want to delete if there is another solution.
+          const root = pixi.import(schema);
```

Comment:
Correspondence for the problem that sprite size at the time of test becomes default value (1, 1).
want to respond if there is an appropriate correction method.

Reply:
ok, let's fix this in another PR.